frontend/packages/kubevirt-plugin/integration-tests-cypress/view/add-source.ts

```diff
@@ -0,0 +1,35 @@
+import { ProvisionSource } from '../../integration-tests/tests/utils/constants/enums/provisionSource';
+
+export const provisionSourceInputs = {
+  [ProvisionSource.URL.getValue()]: 'input[id="provision-source-url"]',
+  [ProvisionSource.CONTAINER.getValue()]: 'input[id="provision-source-container"]',
+};
+
+type DiskSourceOpts = {
+  pvcName: string;
+  pvcNamespace: string;
+};
+
+export const addSource = {
```

Comment:
It needs to take storageClass/accessMode/volueMode into considerations here, in the old tests we extract the values from configMap `KUBEVIRT_STORAGE_CLASS_DEFAULTS`, for simplicity I prefer to just use consts to define them for each storageClass. Either way works for me.

Reply:
UI already sets the correct values. We dont need to change them in tests.

Reply:
Then let's add a step to change the storageClass from the environment variable.

Reply:
Ok, you can specify `STORAGE_CLASS` env variable now.